cobalt/browser/application.cc

```diff
@@ -732,18 +731,13 @@ Application::Application(const base::Closure& quit_closure, bool should_preload,
     options.renderer_module_options.enable_fps_overlay = true;
   }
 
-  if (command_line->HasSwitch(browser::switches::kEnableSkiaRasterizer)) {
-    int enable_skia = 0;
-    base::StringToInt(command_line->GetSwitchValueASCII(
-                          browser::switches::kEnableSkiaRasterizer),
-                      &enable_skia);
-    if (enable_skia) {
-      options.renderer_module_options.rasterizer_type_setting =
-          configuration::Configuration::kSkiaRasterizer;
-    } else {
-      options.renderer_module_options.rasterizer_type_setting =
-          configuration::Configuration::kGlesRasterizer;
-    }
+  bool enable_skia_rasterizer = false;
```

Comment:
Since the point of `enable_skia_rasterizer` and the call to `UseSkiaRasterizer` which toggles `use_skia_rasterizer` essentially serve the same purpose, to propagate this skia flag from `application.cc` to other classes without having to call persistent settings again.

Can we standardize the naming of them between `enable` and `use` so that they are the same?